### PR TITLE
baby's first PR / ports plasmaglass tables from Cit

### PR DESCRIPTION
**Pull request by fighterslam**
## About The Pull Request
 Adds plasma-glass tables to the game with more health than a normal glass table and a purple coloring. 
Sprites by: https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9212

 
## Why It's Good For The Game
 Adds new tables. They look fancy, they're stronger than a normal glass table, and you can stand on them!

## Changelog
:cl: fighterslam, dapnee
add: Adds plasmaglass tables from Citadel
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Is it combustable?

**Comment by fighterslam** [comment]
no, normal plasmaglass isn't flammable why would the table be

**Review by Sinestia — APPROVED** [review]
Checks out and looks fine in-game. 

Most I can even say is that 270 is a strange number as opposed to 250 or 300.  
Does it matter? Definitely not lol.

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -297,6 +298,25 @@
...
+    buildstack = /obj/item/stack/sheet/plasmaglass
+    max_integrity = 270
+    armor = list("melee" = 10, "bullet" = 5, "laser" = 0, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 100)
+
+/obj/structure/table/glass/plasma/Initialize()
+    . = ..()
+    debris += new /obj/item/shard/plasma
+
+/obj/structure/table/glass/plasma/check_break(mob/living/M)
+    return
```
Comment (PowerfulBacon) [inline code/game/objects/structures/tables_racks.dm:318]: should break for fat people only

**Pull request merged**